.circleci/config.yml

```diff
@@ -628,6 +636,70 @@ jobs:
           command: |
             echo "Nightly build run"
 
+  # -------------------------
+  #    JOBS: Windows
+  # -------------------------
+  windows_job:
+    executor: 
+      name: win/default
+    environment:
+      - ANDROID_HOME: "C:\\android-sdk-windows"
+      - ANDROID_NDK: "C:\\android-sdk-windows\\android-ndk-r19c"
+      - ANDROID_BUILD_VERSION: 28
+      - ANDROID_TOOLS_VERSION: 29.0.2
+      - GRADLE_OPTS: -Dorg.gradle.daemon=false
+      - SDK_TOOLS_URL: https://dl.google.com/android/repository/sdk-tools-windows-3859397.zip
+      - NDK_TOOLS_URL: https://dl.google.com/android/repository/android-ndk-r19c-windows-x86_64.zip
+    steps:
+      - restore_cache_checkout:
+          checkout_type: windows 
+      - run:
+          name: Install Android SDK Tools
+          command: choco install android-sdk
+      - run:
+          name: Setup SDK
+          command: sdkmanager --licenses > nul
+      - run: sdkmanager "system-images;android-19;google_apis;armeabi-v7a"
+      - run: sdkmanager "platforms;android-%ANDROID_BUILD_VERSION%"
+      - run: sdkmanager "build-tools;%ANDROID_TOOLS_VERSION%"
+      - run: sdkmanager "add-ons;addon-google_apis-google-23"
+      - run: sdkmanager "extras;android;m2repository"
+      - run:
+          name: Install Android SDK Tools
+          command: choco install android-ndk
+      - restore_cache:
+          keys:
+            - v5-win-node-{{ arch }}-{{ .Branch }}-{{ checksum "yarn.lock" }}
+            - v5-win-node-{{ arch }}-{{ .Branch }}-
+            - v5-win-node-{{ arch }}-
+      - run:
+          name: Install Yarn
+          command: choco install yarn
+      - run:
+          name: Get Env info
+          command: npx envinfo@latest
+      - run:
+          name: Install Dependencies
+          command: yarn install --frozen-lockfile --non-interactive
+      - save_cache:
+          key: v5-win-node-{{ arch }}-{{ .Branch }}-{{ checksum "yarn.lock" }}
+          paths:
+            - C:\Users\circleci\AppData\Local\Yarn
+      - run:
+          name: Flow Check Android
+          command: yarn flow-check-android
+      - run:
+          name: Flow Check iOS
+          command: yarn flow-check-ios
+      - run:
```

Comment:
Consider add `gradlew.bat RNTester:android:app:assembleRelease` here to test windows build.